src/Middleware/Ui.php

```diff
@@ -55,6 +63,11 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
             $session = $GLOBALS['injector']->getInstance(\Horde_Session::class);
             $registry = $GLOBALS['injector']->getInstance(\Horde_Registry::class);
 
+            // getting node information of Horde top-menubar: this will be saved in jsGlobalsHorde
+            $menu = $this->topbar->create('Passwd_Tree_Nodesobject', array('nosession' => true));
```

Comment:
Use ['nosession' => true] instead of array().

Reply:
ok will adapt this